.gitignore

```
@@ -2,7 +2,21 @@
 # Files already tracked by Git are not affected.
 # See: https://git-scm.com/docs/gitignore
 
+# Binaries
+*.exe
+*.exe~
+*.dll
+*.so
+*.dylib
```

Sort lines alphabetically:
```suggestion
*.dll
*.dylib
*.exe
*.exe~
*.so
```

---

@dav3r resolved with [3add000](https://github.com/cisagov/con-pca-tasks/pull/1/commits/3add000073e20deba5ce69e868236d0bde794655)